sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -1329,3 +1366,65 @@ describe("Batching Receiver", () => {
     });
   });
 });
+
+/**
+ * Sets `batchingReceiver` so it's next drain call will result in the connection recycling (via connection.idle()).
```

Comment:
I think these doc comments are no longer accurate since you aren't calling connection.idle()

Reply:
Oops, will update.

Reply:
Updated!